src/Compression/CompressionCodecT64.cpp

```diff
@@ -177,6 +180,8 @@ TypeIndex baseType(TypeIndex type_idx)
         case TypeIndex::Enum16:
         case TypeIndex::Date:
             return TypeIndex::UInt16;
+        case TypeIndex::Date32:
```

Comment:
Let's move this up with `case TypeIndex::Int32`

Reply:
Apart from this, please merge master so `Fast tests` can pass

Reply:
please merge master -- is this already done? I see merge commit in the pr branch.